examples/tagmap/package.json

```diff
@@ -5,16 +5,17 @@
     "build": "NODE_ENV=production webpack --env.prod=true"
   },
   "dependencies": {
-    "axios": "^0.16.2",
     "colorbrewer": "^1.0.0",
     "d3-color": "^1.0.3",
+    "d3-request": "^1.0.5",
     "d3-scale": "^1.0.6",
     "deck.gl": "^4.1.0",
     "luma.gl": "^4.0.1",
-    "rbush": "^2.0.1",
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
-    "react-map-gl": "^3.0.0"
+    "react-map-gl": "^3.0.0",
+    "stats.js": "^0.17.0",
+    "tagmap.js": "^1.1.0"
```

Comment:
So we added this additional dependency. Is this tagmap.js doing what functions in source files deleted in PR do?